### PR TITLE
Add F-Droid badge and shield

### PR DESCRIPTION
**Pull request by thgoebel**
Add F-Droid badge and shield.

If you want a different ordering of badges, let me know, I can amend my commit.

I see you [intentionally removed them](https://github.com/CatimaLoyalty/Android/commit/6516d18cc2d8260b4f3deec6e33604ad0978c831), and I am aware of your parting with F-Droid :/
Still, some people prefer F-Droid over Izzy (e.g. because Izzy does not actually verify RB afaik). So I would prefer the visibility.

### CODE REVIEW

**Comment by TheLastProject** [comment]
Let me make clear that removing this button was not an easy decision.

As an app author, I have to think about keeping my users safe and ensuring they can safely receive app updates. Part of that is making sure I only suggest to them to install the app from distribution channels I can trust to be safe. I sadly have to conclude I can no longer trust F-Droid to be safe. Therefore, I removed the badge, to nudge users more towards IzzyOnDroid. So, the reduced visibility is the point, and I therefore think it's not a good service to my users to re-add the button and give the illusion I recommend them to get my apps from F-Droid. I will, however, not go out of my way to get my apps removed from F-Droid to ensure that users who are stuck with the F-Droid build will keep receiving updates.

I want to make clear that this is not just about my leaving F-Droid. If it was, I would've removed the button months ago. While I am extremely disappointed that nothing changed in F-Droid and there seems to be nobody left who cares to make that organisation more functional, the things that really pushed me over the edge in no longer trusting F-Droid as secure distribution channel for my apps is [the recent failure to fix a reported security issue in fdroidserver](https://github.com/obfusk/fdroid-fakesigner-poc?tab=readme-ov-file) despite a patch being supplied (note: this is an exploit for a security issue that was reported almost a full year before the PoC was made) and the fact that they distributed Catima 2.29.0 [without checking for RB building](https://gitlab.com/fdroid/fdroiddata/-/merge_requests/14905) without my permission. This is on top of a generally very lax attitude towards security throughout the years (I've always been promised issues I named "would be fixed" but to this day that rarely happened and user-contributed security fixes have often been blocked from getting merged by the tech lead for no discernable reason).

Regarding RB: Catima doesn't have true RB in F-Droid, it only has an "RB soft-fail": checking if their APK matches a SHA256sum I publish online during the build (https://gitlab.com/fdroid/fdroiddata/-/commit/642a2de40de022d9a6a4fc2180f1531496600204). This was introduced by me and @obfusk to have some kind of RB check without having to force all users to reinstall as Catima was added to F-Droid way before RB was ever introduced there and uses F-Droid's signature. F-Droid can trivially remove this check at any time and while they did it with my permission for release [2.28.0](https://github.com/CatimaLoyalty/Android/releases/tag/v2.28.0), the fact that I had to go out of my way to get them to re-enable the security for 2.29.0 despite 2.29.0 being RB has futher cemented my distrust in their security hygiene.

To add to the RB discussion, it was @obfusk who noticed that Catima 2.28.0 was not reproducible several days before F-Droid even noticed the build failure. Given there is to my knowledge no RB expert left at F-Droid and the fact that it took the F-Droid team several weeks to start recommending [the fix we documented](https://github.com/CatimaLoyalty/Android/pull/1781) to other projects suffering from the same issue I have my doubts this RB failure would've gotten fixed with just F-Droid's involvement. I'm therefore not very optimistic about F-Droid's ability to keep providing updates for Catima either while the RB soft-fail is there.

Long story short, long term I do not know if F-Droid users will be able to keep safely receiving Catima updates. This uncertainty together with a lack of clear progress within F-Droid to start taking issues more seriously gives me no other choice than to nudge my users to other sources for their security and to distance myself from F-Droid to not get blamed when a security issue from them starts affecting my users.

**Comment by thgoebel** [comment]
Fair enough. Thank you for taking the time to explain it to me. 

I was not aware of the signer poc, thanks for pointing it out.
